lxd/certificates.go

```diff
@@ -615,37 +615,19 @@ func certificatesPost(d *Daemon, r *http.Request) response.Response {
 			return response.BadRequest(fmt.Errorf("Invalid certificate material: %w", err))
 		}
 	} else if req.Token {
-		// Get all addresses the server is listening on. This is encoded in the certificate token,
-		// so that the client will not have to specify a server address. The client will iterate
-		// through all these addresses until it can connect to one of them.
-		addresses, err := util.ListenAddresses(localHTTPSAddress)
+		token, err := createCertificateAddToken(s, req.Name, api.IdentityTypeCertificateClient)
```

Comment:
I suspect here that we don't want to pass in this identity type, since we're not creating a fine-grained TLS identity via this API. 

I would pass an empty string instead, as this API never used that field.